ref/general/requestTiming.adoc

```diff
@@ -22,6 +22,12 @@ For a hung request, apart from printing a warning message along with the relevan
 Three Java cores are generated for a hung request, each a minute apart.
 If a previously hung request is completed, a detailed message is logged, including the ID and time of completion of the request.
 
+A slow request can be slow because the code path length of a request is particularly long or because some resource the request depends on, such as a database is busy.
+Requests that take much longer, and which are therefore likely to be hung, are typically blocked by lack of access to a shared resource, for example, a section of code that only allows serial access and transfers information sequentially.
```

Comment:
this sentence is too long for acrolinx. I think you could break it in two without changing much. Also the "only" should be before "serial".